packages/omd/omd.1.2.0/opam

```diff
@@ -34,6 +34,6 @@ with 0.9.x."""
 flags: light-uninstall
 extra-files: ["omd.install" "md5=0395736894a46f718a77f59ec6fbf1fd"]
 url {
-  src: "http://pw374.github.io/distrib/omd/omd-1.2.0.tar.gz"
-  checksum: "md5=2a1aa5144466a466e33b34de855bcd1c"
+  src: "https://github.com/ocaml/omd/archive/refs/tags/1.2.0.tar.gz"
+  checksum: "sha256=180465a930f27be1bf82bb4709f900cad0d7c69820c0c4efa98573242287637e"
```

Comment:
```suggestion
  checksum: [
    "sha256=180465a930f27be1bf82bb4709f900cad0d7c69820c0c4efa98573242287637e"
    "md5=2a1aa5144466a466e33b34de855bcd1c"
  ]
```

And the same for all of them. Are the tarballs identical? If now, we should try to find them in the cache and upload them on the opam-source-archives repo

Reply:
I hoped them to be the same, but they are different (at least for the ones that I have checked.) That's why I removed the old checksum.

>If now, we should try to find them in the cache and upload them on the opam-source-archives repo

Do you mean if they are not different, we should cache and upload?

I was not able to find archives that have the same hash. I tried https://github.com/ocaml/omd, https://github.com/pw374/omd, and Software Heritage, and they are all different from the recorded hash

Reply:
The cache that I mean is either of

- https://opam.robur.coop/cache/md5/2a/2a1aa5144466a466e33b34de855bcd1c -- this package is not there unfortunately
- https://opam.ocaml.org/cache/md5/2a/2a1aa5144466a466e33b34de855bcd1c -- this package is not there unfortunately

there is a similar URL for the other hashes.

In this case, in the past we marked the package as unavailable since we could not compare the tarball contents with the original ones. I think we should discuss this specific case at the next maintainers meeting

Reply:
Thanks for clarification. Let's discuss this further next week.

Reply:
@mseri do you mind to add it as an item to https://github.com/ocaml/opam-repository/wiki/Meeting-notes? I don't have the permission to modify the wiki page

Reply:
Modified in https://github.com/ocaml/opam-repository/wiki/Meetings-agenda